src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const getFilteredMovies = (movie: Movie, search: string) => {
+  const searchBy = search.toLowerCase();
+  const title = movie.title.toLowerCase();
+  const description = movie.description.toLowerCase();
+
+  return title.includes(searchBy) || description.includes(searchBy);
+};
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer
+    .filter(movies => getFilteredMovies(movies, query));
+
```

Comment:
i think better to save movies in the state, and just update state after filtering

Reply:
agree, let's save movies in state and remove `visibleMovies` variable